internal/bminventory/inventory.go

```diff
@@ -2102,6 +2102,8 @@ func (b *bareMetalInventory) updateClusterData(_ context.Context, cluster *commo
 			&map[string]interface{}{"hyperthreading_enabled": *params.ClusterUpdateParams.Hyperthreading}, usages)
 	}
 
+	b.setUserManagedNetworkingAndMultiNodeUsage(params.ClusterUpdateParams.UserManagedNetworking, cluster.HighAvailabilityMode, usages)
```

Comment:
Similarly to the hyperthreading condition a few lines above this one, we should do the != nil check here and pass a non-pointer value to the `setUserManagedNetworkingAndMultiNodeUsage` function